backend/src/api/cards.js

```diff
@@ -186,3 +186,29 @@ export const unshareCard = async (req, res) => {
     res.status(500).send('Internal server error')
   }
 }
+
+// New endpoint for retrieving cards with tasks
+export const getAllCardsWithTasks = async (req, res) => {
+  try {
+    const projectId = req.params.projectId; 
+    const query = `

```

Comment:
1. Queries should be in `queries.js`
2. I asked you to expand the existed query by getting tasks.
3. This query has another result - not we expected. I mean that we need to add all tasks with card_id = card.id to this card.
Smth like that:
```sql
SELECT 
    c.*, 
    json_group_array(
        json_object(
            'id', t.id,
            'isDone', t.isDone,
            'text', t.text,
            'users', t.users,
            'author', t.author,
            'priority', t.priority,
            'status', t.status
        )
    ) AS tasks
FROM 
    cards c
LEFT JOIN 
    tasks t ON c.id = t.card_id
GROUP BY 
    c.id;
```